app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -249,6 +249,8 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
 
     override fun onResume() {
         val thisActivity = activity
+        settingsPresenter = SettingsPresenter(activity as SkillsActivity)
+        settingsPresenter.onAttach(this)
```

Comment:
This is a workaround. Not a fix

Reply:
Then what can I do for a fix . Please help sir.

Reply:
@iamareebjamal sir by this i will be able to fix login functions if any other malfunctions then?

Reply:
Don't show login if already logged in